### PR TITLE
remove drive letter

### PR DESCRIPTION
**Pull request by hoolymama**
Remove drive letter from maya `-proj` arg. It refers to a location on a linux filesystem and helps Maya to find reference files.

### CODE REVIEW

File: conductor/submitter_maya.py [hunk 1]
```diff
@@ -263,8 +263,9 @@ def generateTasksData(self):
         render_layer_args = ("-rl " + ",".join(render_layers)) if render_layers else ""
 
         # Workspace/Project arg. Only add flag if a workspace has been indicated in the submitter ui
-        workspace = self.extended_advanced_widget.getWorkspaceDir()
-        project_arg = "-proj %s" % file_utils.quote_path(workspace) if workspace.strip() else ""
+        workspace = self.extended_advanced_widget.getWorkspaceDir().strip() or ""
```
Comment (lawschlosser) [inline conductor/submitter_maya.py]: out of curiosity, what condition is the `or ""` accounting for ?

File: conductor/submitter_maya.py [hunk 2]
```diff
@@ -263,8 +263,9 @@ def generateTasksData(self):
...
+        workspace = file_utils.strip_drive_letter(workspace)
+        project_arg = "-proj %s" % file_utils.quote_path(workspace)
```
Comment (lawschlosser) [inline conductor/submitter_maya.py]: what happens if `workspace` is an empty string?

**Comment by hoolymama** [comment]
> I made this change in a PR #238 back in january, but never merged it. I found that it didn't solve the issue that I hoped it would. We should probably should go ahead with it anyway.

Yes, I think so. Without it, Maya can't find references in some situations. Zombiestudios were having problems earlier. If you made the same fix previously then feel free to discard this and merge yours.

**Comment by lawschlosser** [comment]
> > I made this change in a PR #238 back in january, but never merged it. I found that it didn't solve the issue that I hoped it would. We should probably should go ahead with it anyway.
> 
> Yes, I think so. Without it, Maya can't find references in some situations. Zombiestudios were having problems earlier. If you made the same fix previously then feel free to discard this and merge yours.

@hoolymama Ok, I'll go ahead and merge the old PR

**Comment by lawschlosser** [comment]
closed in favor of https://github.com/AtomicConductor/conductor_client/pull/238

**Pull request closed without merging**